test/unit/connection_string.test.ts

```diff
@@ -174,25 +295,25 @@ describe('Connection String', function () {
     context('when the options are provided in the URI', function () {
       context('when the options are equal', function () {
         context('when both options are true', function () {
-          const options = parseOptions('mongodb://localhost/?tls=true&ssl=true');
-
           it('sets the tls option', function () {
+            const options = parseOptions('mongodb://localhost/?tls=true&ssl=true');
```

Comment:
The remainder of changes in this file just pull the parsing into the `it` blocks, so that the parsing doesn't run synchronously when the test suite is loaded.